src/bbsecondary.c

```diff
@@ -139,10 +139,10 @@ bool start_secondary(bool need_secondary) {
     return true;
   //no problems, start X if not started yet
   if (!bb_is_running(bb_status.x_pid)) {
-    char pci_id[13];
+    char pci_id[16];
     static char *x_conf_file;
     // 0-255 bus, 0-31 slot, 0-7 func
-    snprintf(pci_id, 13, "PCI:%03d:%02d:%o", pci_bus_id_discrete->bus,
+    snprintf(pci_id, 16, "PCI:%03d:%02d:%o", pci_bus_id_discrete->bus,
```

Comment:
Would have been nice to use `sizeof pci_id` instead of `16` here (like it was already the case in other places).

Reply:
Thanks, fixed